src/Ads/index.js

```diff
@@ -16,223 +16,77 @@
-          delete timeEvents.firstQuartile
-          sendBeacon(ad.callbacks, 'firstQuartile')
-        }
-        if (timeEvents && timeEvents.midPoint && videoEl.currentTime >= timeEvents.midPoint) {
-          fireOnConsumer('MidPoint', ad)
-          delete timeEvents.midPoint
-          sendBeacon(ad.callbacks, 'midPoint')
-        }
-        if (
-          timeEvents &&
-          timeEvents.thirdQuartile &&
-          videoEl.currentTime >= timeEvents.thirdQuartile
-        ) {
-          fireOnConsumer('ThirdQuartile', ad)
-          delete timeEvents.thirdQuartile
-          sendBeacon(ad.callbacks, 'thirdQuartile')
-        }
-      },
-      stalled() {
-        fireOnConsumer('Stalled', ad)
-        timeout = setTimeout(() => {
-          cleanup()
-        }, 5000) // make timeout configurable
-      },
-      canplay() {
-        timeout && clearTimeout(timeout)
-      },
-      error() {
-        fireOnConsumer('Error', ad)
-        cleanup()
-      },
-      // this doesn't work reliably on sky box, moved logic to timeUpdate event
-      // loadedmetadata() {
-      //   // calculate when to fire the time based events (now that duration is known)
-      //   timeEvents = {
-      //     firstQuartile: videoEl.duration / 4,
-      //     midPoint: videoEl.duration / 2,
-      //     thirdQuartile: (videoEl.duration / 4) * 3,
-      //   }
-      // },
-      abort() {
-        cleanup()
-      },
-      // todo: pause, resume, mute, unmute beacons
-    }
-    // add all listeners
-    Object.keys(handlers).forEach(handler => videoEl.addEventListener(handler, handlers[handler]))
-
-    videoEl.play()
-  })
+let getConfig = function() {
+  return {		
+    siteSection: '123',		
+    profile: '123'		
+  }
 }
 
-const sendBeacon = (callbacks, event) => {
-  if (callbacks && callbacks[event]) {
-    Log.info('Ad', 'Sending beacon', event, callbacks[event])
-    return callbacks[event].reduce((promise, url) => {
-      return promise.then(() =>
-        fetch(url)
-          // always resolve, also in case of a fetch error (so we don't block firing the rest of the beacons for this event)
-          // note: for fetch failed http responses don't throw an Error :)
-          .then(response => {
-            if (response.status === 200) {
-              fireOnConsumer('Beacon' + event + 'Sent')
-            } else {
-              fireOnConsumer('Beacon' + event + 'Failed' + response.status)
-            }
-            Promise.resolve(null)
-          })
-          .catch(() => {
-            Promise.resolve(null)
-          })
-      )
-    }, Promise.resolve(null))
-  } else {
-    Log.info('Ad', 'No callback found for ' + event)
+let getPolicy = function() {
+  return {		
+    adSkipTier: 'NOSKIP_NORMAL_SPEED',		
+    adSkipGracePeriodSeconds: 60		
   }
 }
 
-const fireOnConsumer = (event, args) => {
-  if (consumer) {
-    consumer.fire('$ad' + event, args)
-    consumer.fire('$adEvent', event, args)
+let getPrivacy = function() {
```

Comment:
we need to work on the output format here, it needs to comply with some industry standards as badger does, I think.